kernel/posixtest.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env python
+
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+#
+# Copyright: 2016 IBM
+# Author: Praveen K Pandey <praveen@linux.vnet.ibm.com>
+#
+# Based on code by Mohammed Omar <mohd.omar@in.ibm.com>
+#   copyright: 2008 IBM
+#   https://github.com/autotest/autotest-client-tests/tree/master/posixtest
+
+import os
+
+from avocado import Test
+from avocado import main
+from avocado.utils import archive
+from avocado.utils import process
+from avocado.utils import build
+from avocado.utils.software_manager import SoftwareManager
+
+
+class Posixtest(Test):
+
+    '''
+     posix test  provides conformance, functional, and stress testing on Os Threads, Clocks & Timers, Signals, Message Queues, and Semaphores.
```

Comment:
You can wrap this line with 72 characters (https://www.python.org/dev/peps/pep-0008/#maximum-line-length)